com.insynchq.Insync.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.insynchq.Insync
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base-version: "24.08"
+command: /app/usr/lib/insync/insync
+separate-locales: false
+rename-app-id: com.insynchq.Insync
+rename-desktop-file: insync.desktop
+rename-icon: insync
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+    # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.kde.StatusNotifierItem
+  - --own-name=org.kde.*
+  # Needed for /dev/disk/by-id which insync accesses
+  - --device=all
+  - --filesystem=~/Insync
+  - --persist=.config
+  - --filesystem=xdg-config/autostart:create
+build-options:
+  strip: true
+modules:
+  - name: insync
+    buildsystem: simple
+    build-commands:
+      - ar x insync_*.deb
+      - rm insync_*.deb
+      - tar xf data.tar.gz
+      - mv usr $FLATPAK_DEST
+      - cp -r $FLATPAK_DEST/usr/share $FLATPAK_DEST
+      - desktop-file-edit --set-key=Exec --set-value="run.sh" $FLATPAK_DEST/share/applications/insync.desktop
+      - install -Dm644 com.insynchq.Insync.appdata.xml $FLATPAK_DEST/share/metainfo/com.insynchq.Insync.metainfo.xml
+      - install -Dm755 run.sh $FLATPAK_DEST/bin/run.sh
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://cdn.insynchq.com/builds/linux/3.9.6.60027/insync_3.9.6.60027-plucky_amd64.deb
```

Comment:
is this redistributable? If not then it should be extra-data.

Reply:
Given your answer (or lack thereof) to https://github.com/flathub/flathub/pull/6738#discussion_r2216299830 it's **not redistributable** (the default).